resources/js/components/attribute/Tabular.vue

```diff
@@ -35,183 +36,37 @@
-                            @change="e => updateDirtyState(e, $index, column.id)"
-                        />
-
-                        <daterange-attribute
-                            v-else-if="column.datatype == 'daterange'"
-                            :ref="el => setRef(el, `${$index}_${column.id}`)"
-                            :disabled="disabled || row.mark_deleted"
-                            :name="`${name}-column-attr-${column.id}`"
-                            :value="row[column.id]"
-                            @change="e => updateDirtyState(e, $index, column.id)"
-                        />
-
-                        <singlechoice-attribute
-                            v-else-if="column.datatype == 'string-sc'"
-                            :ref="el => setRef(el, `${$index}_${column.id}`)"
-                            :disabled="disabled || row.mark_deleted"
-                            :name="`${name}-column-attr-${column.id}`"
-                            :value="row[column.id]"
-                            :selections="state.selections[column.id]"
-                            @change="e => updateDirtyState(e, $index, column.id)"
-                        />
-
-                        <multichoice-attribute
-                            v-else-if="column.datatype == 'string-mc'"
-                            :ref="el => setRef(el, `${$index}_${column.id}`)"
-                            :disabled="disabled || row.mark_deleted"
-                            :name="`${name}-column-attr-${column.id}`"
-                            :value="row[column.id]"
-                            :selections="state.selections[column.id]"
-                            @change="e => updateDirtyState(e, $index, column.id)"
-                        />
-
-                        <userlist-attribute
-                            v-else-if="column.datatype == 'userlist'"
-                            :ref="el => setRef(el, `${$index}_${column.id}`)"
-                            :name="`${name}-column-attr-${column.id}`"
-                            :value="row[column.id]"
-                            @change="e => updateDirtyState(e, $index, column.id)"
-                        />
-
-                        <url-attribute
-                            v-else-if="element.datatype == 'url'"
-                            :ref="el => setRef(el, `${$index}_${column.id}`)"
-                            :name="`${name}-column-attr-${column.id}`"
-                            :value="row[column.id]"
-                            @change="e => updateDirtyState(e, $index, column.id)"
-                        />
-                    </td>
-                    <td
-                        v-if="!disabled"
-                        class="text-center"
+                        v-if="row.hidden_info"
+                        class="text-muted text-center fs-5 p-2 bg-primary-subtle"
+                        :colspan="Object.keys(state.columns).length + 2"
```

Comment:
use computed at this and the next position? (redundancy)

Reply:
👍